app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -318,7 +327,11 @@ private NagiosExtension convertToEbeanNagiosExtension(final models.internal.Nagi
          */
         @Override
         public PagedList<models.ebean.Alert> createAlertQuery(final AlertQuery query) {
+            if (query.getOrganization() == null) {
```

Comment:
Sorry I didn't get a chance to call back. I sent you an email on this with some of my thoughts.